elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -314,20 +312,28 @@ class PfeNavigation extends PFElement {
    * Almost all open/close actions should go through this._changeNavigationState, not this function
    *
    * @param {object} toggleElement Toggle Button DOM Element
+   * @param {object} toggleId Toggle Button DOM Element ID attribute
```

Comment:
Confused here, `toggleId` isn't a param for the fuction? This could work as a code comment above the var declaration?

Reply:
@wesruv oh oops meant to move that down as a comment above the var, thanks for catching that. I will get that updated.